leny62/Personal-webs
# ft(contact page) : connected contact page to Cloud firestore

**What does this PR do**
It connects Contact page to cloud Firestore.
It shows a table of inquiries on the admin dashboard.

**Feature Description**
An admin will be able to read the sent queries from his dashboard.
Any user will be able to contact the admin from the contact page.

***How to manually test this***
Got contact page.
write the message, and add a name and email.
Go to login page and login a signed up account.
Go to messages page.
Admin will see all enquiries displayed on table.
<img width="1440" alt="Screen Shot 2020-10-11 at 18 11 12" src="https://user-images.githubusercontent.com/68828382/95683730-6aafe880-0bed-11eb-8356-3a6e318939b7.png">
